### ISSUE TITLE
Split player colours out of control.lua

### ISSUE THREAD

**Issue by plague006** [Valansch/RedMew#195]
Split player colours out of control.lua.

Make it a patreon (and admin?) feature only.

**Comment by Valansch** [comment]
All players have access to all features, except admins features obviously. Especially QoL features like this.

**Comment by plague006** [comment]
Agreed not in code and in a file like regulars. I would suggest not tying it to the /color command though and instead creating something distinct like /save-color since, when I asked around, a surprising number of people like getting random colours per map.

**Comment by Valansch** [comment]
Regular is saved to a database like file. It is not meant to be used by adding users to it.
Unfortunedly this only works on the diffie server, so currently poor @grilledham updates the list manually. Color should and would use the same system, reguardless if we make it tie into /colors or not.

>I would suggest not tying it to the /color command though and instead creating something distinct like /save-color since, when I asked around, a surprising number of people like getting random colours per map.

/colors is not random tho. We can instead do a /deleteColor command that removes the user. I would guess more users want their color to be persistent than the ones that don't. But at this point thats just my opinion^^

**Comment by grilledham** [comment]
I must of missed this issue yesterday. 

While I agree with everything Valansch said. In the mean time I did add a player colour's module, with the colors hard coded, because I like having my color auto set and other people probably would too. And I didn't like feeling that I had special treatment in the code base.

It would probably be best not to advertise this feature to the wider player base as I don't fancy manually updating people's colors on request. But I'll make exception for admins, or if anyone makes a PR with their colors. I already added your colors plague006.

But the end goal is to ultimately remove this and use the same system as we do for regulars on Diffie's server. And it should definitely be a feature that is open to everyone.
